examples/impute/plot_iterative_imputer_variants_comparison.py

```diff
@@ -97,19 +95,38 @@
 # with different estimators
 estimators = [
     BayesianRidge(),
-    DecisionTreeRegressor(max_features="sqrt", random_state=0),
-    ExtraTreesRegressor(n_estimators=10, random_state=0),
+    RandomForestRegressor(
+        # We tuned the hyperparameters of the ExtraTreesRegressor to minimize
+        # the execution time
+        n_estimators=4,
+        max_depth=10,
+        max_samples=0.5,
+        n_jobs=2,
+        random_state=0,
+    ),
+    ExtraTreesRegressor(
+        # We tuned the hyperparameters of the ExtraTreesRegressor to minimize
+        # the execution time
+        n_estimators=4,
+        max_depth=10,
+        max_samples=0.5,
+        n_jobs=2,
+        random_state=0,
+    ),
     KNeighborsRegressor(n_neighbors=15),
 ]
 score_iterative_imputer = pd.DataFrame()
-for impute_estimator in estimators:
+tolerances = (1e-3, 1e-1, 1e-1, 1e-2)
```

Comment:
It would be nice to have a small comment mentioning that the iterative imputer is sensible to the tolerance and dependent to the estimator used internally.